### DISCUSSION TITLE
How can I get the list of registered events in a Node or Process?

### DISCUSSION THREAD

**Discussion by pmbanugo** [ergo-services/ergo#199] [Q&A]
Is there a way to retrieve the list of registered events (and probably their token as well) from a Node or Process? I couldn't find that in the docs and I think that'll be nice to have, making it easy to retrieves tokens or know what events are already created

**Comment by halturin** [comment]
`gen.Process` provides `Events` method to get the list of registered events. Tokens are private data so its not a good idea to share them.
